service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/service/ReportService.java

```diff
@@ -1,37 +1,40 @@
 package uk.nhs.adaptors.oneoneone.cda.report.service;
```

Comment:
Shouldn't we rename this to EncounterService?

Reply:
Good idea. Done.